docs/README.md

```diff
@@ -88,6 +88,26 @@ For example:
 },
 ```
 
+Example of using some variables in the `product-selling-price` block to render unit multiplier and measurement unit only on products that have it:
+
+```json
+{
+  "product-selling-price": {
+    "props": {
+      "message": "{sellingPriceValue}{hasMeasurementUnit, select, true { / {hasUnitMultiplier, select, true {{unitMultiplier}} false {}} {measurementUnit}} false {}}"
+    }
+  }
+}
+```
+
+On products that have measurement unit and unit multiplier, it will render something like:
+
+> $24.00 / 2 oz
+
+On products that doesn't, it will render only the price:
+
+> $24.00
```

Comment:
```suggestion
You can use the `product-selling-price`'s `sellingPriceValue`, `hasMeasurementUnit`, `unitMultiplier`, and `measurementUnit` variables together to render unit measurement and multiplier on products that are sold per unit. 

For example:

json
{
  "product-selling-price": {
    "props": {
      "message": "{sellingPriceValue}{hasMeasurementUnit, select, true { / {hasUnitMultiplier, select, true {{unitMultiplier}} false {}} {measurementUnit}} false {}}"
    }
  }
}

According to the example above, products sold per unit would be rendered as follows:

$24.00 / 2 oz

> ℹ️ *Notice that the unit measurement and multiplier would be properly rendered alongside their price*

Still, according to the example, products that are not sold per unit would render only their price:

$24.00
```

Reply:
I was not able to submit my comment using ``` to wrap the code and the price examples ($24.00 / 2 oz and $24.00). Please, commit your changes using them anyways hehe

Reply:
Also, could you please add this new piece of info in the 3rd step, below the `product-selling-price` topic? :))